internal/jujuapi/streamproxy_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 Canonical.
+
+package jujuapi_test
+
+import (
+	"context"
+
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/common"
+	gc "gopkg.in/check.v1"
+
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/openfga"
+	ofganames "github.com/canonical/jimm/v3/internal/openfga/names"
+)
+
+type streamProxySuite struct {
+	websocketSuite
+}
+
+var _ = gc.Suite(&streamProxySuite{})
+
+func (s *streamProxySuite) TestDebugLogs(c *gc.C) {
+	conn := s.open(c, &api.Info{ModelTag: s.Model.ResourceTag()}, "bob")
+	defer conn.Close()
+	_, err := common.StreamDebugLog(context.TODO(), conn, common.DebugLogParams{})
+	c.Assert(err, gc.IsNil)
```

Comment:
I don't think I fully understand this test, does no error really mean its working?

Reply:
It actually does funnily enough, because it tests that we can go all the way from the client through to the Juju controller without any auth errors, etc. It would be better to also test there are some debug logs, but I was always getting an empty list of debug logs (probably because there is no application deployed but I'll take another look at that).

Reply:
Yeah let's add an app and try get some logs somehow... If it's super difficult, leave a comment explaining why we aren't testing any logs coming through please.

Reply:
Deploying an app seems tricky, I took a brief look at the [Deploy Client](https://github.com/juju/juju/blob/3.5/api/client/application/client.go#L126) and I don't want to deploy an actual charm from charmhub in these tests. So I've opted to leave a comment.

Reply:
Sure sure